controller/filter.go

```diff
@@ -37,6 +37,7 @@ const (
 	filterKeyWorkItemType = "workitemtype"
 	filterKeyState        = "state"
 	filterKeyLabel        = "label"
+	filterKeyTitle        = "title"
```

Comment:
Maybe we should start outsourcing this as its own `type FilterType string`.

Reply:
As of now, this is the only place where it is used.  Once we find another place, we can outsource it.